index.html

```diff
@@ -0,0 +1 @@
+<meta http-equiv="refresh" content="URL=https://austinbwanika.github.io/ugandaemr-technicalguide/SUMMARY.html" />
```

Comment:
@AustinBwanika Remove "https://austinbwanika.github.io/ugandaemr-technicalguide/" from the URL

Reply:
@ssmusoke I have removed the specified part but the site now comes out as blank page. what does that mean?

Reply:
Because you changed the file name to SUMMARY.md which I did not tell you to do

Reply:
@ssmusoke sorry for that I have now changed it to SUMMARY.html

Reply:
I told you to change the name of the file in your link, not the name of the file, do not rush as you are now wasting time